runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -915,6 +915,21 @@ private Debuggee registerDebuggee(CloudDebugger debuggerClient, String uniquifie
 
   @Override
   public DataflowPipelineJob run(Pipeline pipeline) {
+
+    if (useUnifiedWorker(options)) {
```

Comment:
I reverted every commit that was red, and then added just this. It still fails in the same way. That's interesting...

Reply:
Nevermind, I was looking at stale results. It succeeds. Since the failures were in branches for `!useUnifiedWorker` this should not even execute. So the failure really had _something_ to do with it being a `Set`.